packages/collector/src/agentConnection.js

```diff
@@ -377,6 +377,11 @@ function sendData(path, data, cb, ignore404) {
           statusCodeError.statusCode = res.statusCode;
           cb(statusCodeError);
           return;
+        } else {
+          const statusCodeError = new Error(res.statusMessage);
```

Comment:
This `else` branch looks like it breaks the `ignore404` mechanism. Is this still needed now that the OTel exporter uses `backend_conector` from `@instana/serverless`? What's the reason behind this change?

Reply:
I will remove the PR as this is WIP and many things are not yet concluded

Reply:
But it can be removed. I thought this was something that was missing in the code anyways, because `sendData` gives a callback with no errors when there was actually an error.
I will just rollback the changes.